MiniApp/Classes/Storage/MiniAppPreferences.swift

```diff
@@ -0,0 +1,15 @@
+class MiniAppPreferences {
```

Comment:
How about the name `MiniAppStatus` instead? This matches the name on Android and more clearly communicates what this class does.

Reply:
Done